src/containers/modals/deposit/steps/ApproveStep.js

```diff
@@ -130,7 +130,7 @@ function ApproveStep ({
       {allowance && allowance !== '0' && allowance !== weiAmount.toString() && (
         <>
           <p>
-            {`You are already approved to deposit ${logAmount(allowance, tokenInfo.decimals)} ${tokenName}. Since this amount does not match how much you want to deposit now, you will need to reset your allowance.`}
+            {`You are already approved to deposit ${logAmount(allowance, tokenInfo.decimals)} ${tokenName}. Since this amount does not match the ${value} ${tokenName} you want to deposit now, you will need to reset your allowance.`}
```

Comment:
`You are already approved to deposit ${logAmount(allowance, tokenInfo.decimals)} ${tokenName}. Since this amount does not match the ${value} ${tokenName} you want to deposit now, you will need to reset your allowance.` ->

`The ${value} ${tokenName} deposit doesn't match the deposit you've already approved (${logAmount(allowance, tokenInfo.decimals)} ${tokenName}). To continue with your deposit, you need to reset your allowance first.`

Reply:
Btw, what do you mean by allowance in this particular case?

Reply:
so you have to call `approve` on x amount, which is then your `allowance` of how much you can deposit. so if you approve 10 weth, you are allowed to deposit up to 10 weth. but if you then want to deposit 11 weth, you have to re-approve to be allowed to deposit up to 11 weth.